src/utils/apicalls.js

```diff
@@ -0,0 +1,21 @@
+export const fetchCardList = (url) => {
+	return fetch(url)
+	.then(response => {
+		if(!response.ok) {
+			throw Error('Error fetching data')
```

Comment:
You'll notice on my PR I wrote my `throw Error` incorrectly, 
I'll fix that on my next push